docs/pywbemclicmdshelp.rst

```diff
@@ -1948,11 +1950,13 @@ The following defines the help output for the `pywbemcli server get-centralinsts
       -o, --organization ORG-NAME     Filter by the defined organization. (ex. -o
                                       DMTF
       -p, --profile PROFILE-NAME      Filter by the profile name. (ex. -p Array
-      -c, --central-class CLASSNAME   Optional. Required only if profiles supports
+      --cc, --central-class CLASSNAME
+                                      Optional. Required only if profiles supports
                                       only scopig methodology
-      -s, --scoping-class CLASSNAME   Optional. Required only if profiles supports
+      --sc, --scoping-class CLASSNAME
+                                      Optional. Required only if profiles supports
                                       only scopig methodology
-      -S, --scoping-path CLASSLIST    Optional. Required only if profiles supports
+      --sp, --scoping-path CLASSLIST  Optional. Required only if profiles supports
                                       only scopig methodology. Multiples allowed
       -r, --reference-direction [snia|dmtf]
                                       Navigation direction for association.
```

Comment:
docs/pywbemclicmdshelp.rst is out of date, which is th ereason it shows `-r` on `--reference-direction`.

Reply:
DONE